src/bots/mascot-bot.js

```diff
@@ -39,6 +46,20 @@ class MascotBot extends SlackBot {
     });
   }
 
+  setTopic(channel, topic, isPublic = true) {
```

Comment:
Setting the topic for a public room (channel) uses a different API call than a private room (group). I wanted to normalize the API call used by behavior developers.